### ISSUE TITLE
Dataexplorer throws error on sorting a numeric identifier

### ISSUE THREAD

**Issue by bartcharbon** [molgenis/molgenis#2157]
If the identifier of a dataset is specified as decimal or int in the metadata the dataexplorer gives you an error if you try to sort on this column.

Either the importer should not allow numeric identifiers or this should be fixed in the dataexplorer

**Comment by erwinwinder** [comment]
This was already fixed

**Comment by fdlk** [comment]
Ran into this today on molgenis01.

**Comment by fdlk** [comment]
```
Error! Failed to execute phase [query_fetch], all shards failed; shardFailures {[Hmlmo97eRGmJQPYgiAHhHg][molgenis][0]: QueryPhaseExecutionException[[molgenis][0]: query[ConstantScore(cache(+_type:org_molgenis_test_TypeTest +org.elasticsearch.index.search.nested.NonNestedDocsFilter@1cf91110))],from[0],size[20],sort[]: Query Failed [Failed to execute main query]]; nested: ElasticsearchException[java.lang.NumberFormatException: Invalid shift value in prefixCoded bytes (is encoded value really an INT?)]; nested: UncheckedExecutionException[java.lang.NumberFormatException: Invalid shift value in prefixCoded bytes (is encoded value really an INT?)]; nested: NumberFormatException[Invalid shift value in prefixCoded bytes (is encoded value really an INT?)]; }
```

**Comment by fdlk** [comment]
Magically fixed (again) in 1.11!
